c/src/ml-api-inference-single.c

```diff
@@ -979,16 +980,25 @@ ml_single_open_custom (ml_single_h * single, ml_single_preset * info)
   for (i = 0; i < num_models; i++)
     g_strstrip (list_models[i]);
 
-  status = _ml_validate_model_file ((const char **) list_models, num_models,
-      &nnfw);
-  if (status != ML_ERROR_NONE) {
-    _ml_error_report_continue
-        ("Cannot validate the model (1st model: %s. # models: %d). Error code: %d",
-        list_models[0], num_models, status);
-    g_strfreev (list_models);
-    return status;
+  /**
+   * Note : OpenVINO and flare use the bin extension.
```

Comment:
well, I suggest updating _ml_validate_model_file() function. plz do not update this block.
We simply check file extension and pass the validation.
```
int _ml_validate_model_file()
{
...
  /** @todo Make sure num_models is correct for each nnfw type */
  switch (*nnfw) {
...
    case ML_NNFW_TYPE_FLARE:
      if (!g_str_equal (file_ext[0], ".bin")) {
        status = ML_ERROR_INVALID_PARAMETER;
      }
      break;
}
```

Reply:
Currently ML_NNFW_TYPE_FLARE is not included ml_nnfw_type_e.
It is defined to avoid ACR.
#define ML_NNFW_TYPE_FLARE 23
when code is changed

```
../c/src/ml-api-inference-single.c: In function ‘_ml_validate_model_file’:
../c/src/ml-api-inference-single.c:2051:5: error: case value ‘23’ not in enumerated type ‘ml_nnfw_type_e’ [-Werror=switch]
 2051 |     case ML_NNFW_TYPE_FLARE:
      |     ^~~~
```

Reply:
So I will change the code as below.
```
  /** @todo Add checking ML_NNFW_TYPE_FLARE to below switch */
  if (*nnfw == ML_NNFW_TYPE_FLARE) {
    if (!g_str_equal (file_ext[0], ".bin")){
      _ml_error_report("Flare accepts .bin file only. Please support correct file extension. You have specified: \"%s\"", file_ext[0]);
      status = ML_ERROR_INVALID_PARAMETER;
    }
    goto done;
  }
```